.eslintrc

```diff
@@ -56,8 +58,82 @@
       "requireParamDescription": false
     }],
 
+    // Airbnb overwrites
```

Comment:
overwriting all the rules that are currently failing from airbnb's eslint. i'll go over each rule and fix them individually if we want that particular rule